db/migrate/20201012165117_create_posts_tags_join_table.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePostsTagsJoinTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts_tags, id: false do |t|
+      t.bigint :post_id
+      t.bigint :tag_id
+    end
+
+    add_index :posts_tags, :post_id
+    add_index :posts_tags, :tag_id
```

Comment:
You might want a unique index across both ids:

```ruby
add_index :posts_tags, [:post_id, :tag_id], unique: true
```

This will avoid duplicate records in the join table and speed up lookups.